### PR TITLE
Update the governance page

### PR DESCRIPTION
**Pull request by svekars**

### CODE REVIEW

**Comment by ezyang** [comment]
Deferring to @orionr

File: docs/source/community/governance.rst [hunk 1]
```diff
@@ -1,128 +1,292 @@
...
+* Light criteria of moving module maintenance to ‘emeritus’
+  status if they don’t actively participate over long periods
+  of time. Each module maintainer group may define the inactive
+  period that’s appropriate for that module.
+* The membership is for an individual, not a company.
+
+The Process for Nomination
+++++++++++++++++++++++++++
+
+* We have a `nomination form <TBA>`__, where anyone in the community can
```
Comment (orionr) [inline docs/source/community/governance.rst:133]: Should we change this to `We will have a nomination form, where...` and no link for now? @b0noI we are still blocked on the form, correct? Thanks.

File: docs/source/community/governance.rst [hunk 2]
```diff
@@ -1,128 +1,292 @@
...
+* The core maintainers then evaluate all information and make
+  a final decision to Confirm or Decline the nomination. The
+  decision of the core maintainers has to be articulated well
+  and would be public.
+
+The Process for Removal
++++++++++++++++++++++++
+
+* Similar to the process for nomination, anyone in the community
+  can `nominate a person to be removed <TBA>`__ from a Module maintainer
```
Comment (orionr) [inline docs/source/community/governance.rst:157]: Same, but just skip the link for now?

**Review by orionr — APPROVED** [review]
Let's remove the TBA links but otherwise looks good!

**Review by b0noI — APPROVED** [review]

File: docs/source/community/governance.rst (hunk 1 above)
Comment (orionr) [reply docs/source/community/governance.rst:133]: @svekars can you update to the above suggestion please and then we can land this? Thanks!

**Review by orionr — CHANGES_REQUESTED** [review]
Let's be explicit for those changes needed.

**Review by orionr — APPROVED** [review]
LGTM! Thanks.

**Pull request closed without merging**